org.osgi.test.cases.jdbc/src/org/osgi/test/cases/jdbc/junit/JDBCTestCase.java

```diff
@@ -57,127 +74,155 @@ private static class DfltProps extends Properties {
 		}
 	}
 
-	private static final String	FILTER_CAPABILITY_CONNECTIONPOOLDATASOURCE	= "("
-			+ OSGI_JDBC_CAPABILITY + "="
-			+ OSGI_JDBC_CAPABILITY_CONNECTIONPOOLDATASOURCE + ")";
-
-	private static final String	FILTER_CAPABILITY_DATASOURCE					= "("
-			+ OSGI_JDBC_CAPABILITY + "=" + OSGI_JDBC_CAPABILITY_DATASOURCE
-			+ ")";
-
-	private static final String	FILTER_CAPABILITY_DRIVER						= "("
-			+ OSGI_JDBC_CAPABILITY + "=" + OSGI_JDBC_CAPABILITY_DRIVER + ")";
-
-	private static final String	FILTER_CAPABILITY_XADATASOURCE					= "("
-			+ OSGI_JDBC_CAPABILITY + "=" + OSGI_JDBC_CAPABILITY_XADATASOURCE
-			+ ")";
-
-	static String				databaseName									= "dbName";
-	static String				dataSourceName									= "dsName";
-	static String				description										= "desc";
-	static String				password										= "pswd";
-	static String				user											= "usr";
-
-	@Test
-	public void testMinimumOneDataSourceFactory(
-			@InjectService(cardinality = 1, filter = "("
-					+ OSGI_JDBC_DRIVER_CLASS + "=*)")
-			ServiceAware<DataSourceFactory> sa) {
-		assertThat(sa.getService()).isNotNull();
-
-		ServiceReferenceAssert.assertThat(sa.getServiceReference())
-				.hasServicePropertiesThat()
-				.containsKey(OSGI_JDBC_CAPABILITY)
-				.extractingByKey(OSGI_JDBC_CAPABILITY)
-				.isNotNull();
-
-	}
-
+	private static final List<JdbcCapability<?>> ALL_CAPABILITIES = List.of(
```

Comment:
We still need to support Java 8. So we cannot use List.of. I am surprise you do not get a compilation error here.